client/src/components/PickDate/PickDate.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { DateInput } from '@blueprintjs/datetime';
+import './PickDate.scss';
+
+class PickDate extends React.Component {
+  state = {
+    closeOnSelection: true,
+    date: null,
+    disabled: false
+  };
+
+  handleDateChange = date => {
+    this.setState({ date: new Date(date) });
+  };
+  render() {
+    return (
+      <div>
+        <div className="content">
+          <span> DueDate: </span>
+        </div>
+        <DateInput
+          formatDate={date => date.toLocaleString()}
+          onChange={this.handleDateChange}
+          parseDate={str => new Date(str)}
+          placeholder={'M/D/YYYY'}
```

Comment:
Instead of 'M/D/YYYY', use 'MM/DD/YYYY'

Also, we should be using DD/MM/YYYY format, not MM/DD/YYYY.